en/contributing/build.md

```diff
@@ -220,17 +220,22 @@ The main steps are:
    cd DroneCore
    git submodule update --init --recursive
    ```
-1. Enter the following command in your host's terminal:
-   ```sh
-   docker run --rm -it -v $HOME/<path-to-dronecore-repo>/DroneCore:/home/docker1000/src/DroneCore:rw dronecore/dronecore bash
-   ```
+1. Enter either of the following command in your host's terminal:
+   * Fedora
+     ```sh
+     docker run --rm -it -v $HOME/DroneCore:/home/docker1000/src/DroneCore:rw dronecore/dronecore-fedora-27 bash
+     ```
+   * Ubuntu
+     ```sh
+     docker run --rm -it -v $HOME/DroneCore:/home/docker1000/src/DroneCore:rw dronecore/dronecore-ubuntu-16.04 bash
+     ```
    > **Note** The `-v` flag maps a directory on your host (left side) to a path in
    the container (right side). Above you need to specify the left-side path to the DroneCore repository on your host. The container path must be set as above.
 
-   > **Note** The `-v` flag maps a directory on your host (left side) to a path in the container (right side).  You need to specify the left-side path to the DroneCore repository on your host and the container path must be set as above.
+   > **Note** The `-v` flag maps a directory on your host (left side) to a path in the container (right side).  You need to specify the left-side path to the DroneCore repository on your host and the container path must be set as above. 
    > The `--rm` automatically cleans up leftover docker containers after you exit the docker container.
 
-   Docker will download an image from [Docker Hub](https://hub.docker.com/r/dronecore/dronecore/), use it to create a container, and then open a bash prompt:
+   Docker will download an image from [Docker Hub](https://hub.docker.com/u/dronecore/), use it to create a container, and then open a bash prompt:
    ```
    root*81ebe14d0c1a:/home/docker1000/src/Dronecore#
```

Comment:
So yes, I don't get this mapping - instead I get something like: `[root@06e7da84c095 DroneCore]#`